### ISSUE TITLE
type of battery

### ISSUE THREAD

**Issue by mooncode30** [pybamm-team/PyBaMM#2919]
can phybamm be used on lithium-polymer battery models?

**Comment by tinosulzer** [comment]
Probably. Do you have a reference with the model and parameters you are trying to implement?

**Comment by tinosulzer** [comment]
Yes, but you will need to give more details about the battery chemistry and find a literature reference for the parameters describing the polymer electrolyte.

**Comment by mooncode30** [comment]
can battery lithium ion charge and discharge data with the CC CV method be generated directly in excel?
